src/Carousel.tsx

```diff
@@ -724,6 +725,7 @@ class Carousel extends React.Component<CarouselProps, CarouselInternalState> {
           <ul
             ref={this.listRef}
             className={`react-multi-carousel-track ${sliderClass}`}
+            dir={this.props.isRTL ? 'rtl' : 'ltr'}
```

Comment:
Please don't force this through direction. I have an issue where in RTL mode for browsers this doesn't render correctly at all (broken for Arabic and Hebrew websites).